driver/k8s/node_service.go

```diff
@@ -68,7 +69,11 @@ func (s NodeService) GetCapacityByTopologyLabel(ctx context.Context, topology, d
 			if v != topology {
 				continue
 			}
-			return s.extractCapacityFromAnnotation(&node, dc)
+			c, err := s.extractCapacityFromAnnotation(&node, dc)
+			if err != nil {
+				return 0, ErrDeviceClassNotFound
```

Comment:
In this function, there is no device class check, and it is actually in extractCapacityFromAnnotation.
Would you move this code to extractCapacityFromAnnotation?